git/datadog_create_test.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+api_key="DATADOG-APIKEY"
```

Comment:
Where is this going to come from?

Reply:
This has to be added manually from user's account

Reply:
So I should conclude that this script is edited manually, then somehow manually injected into DataDog via some kind of web interface?

Reply:
also, it would be nice not to encourage the script user to put his/her creds in plain text in a file. I prefer to have the script take input vars or leverage loaded env vars.